test/image_test.py

```diff
@@ -1016,6 +1030,13 @@ def test_image_cmd_nonempty(builder_version, servicer, client, tmp_path_with_con
         assert 'CMD ["echo", "hello"]' in layers[0].dockerfile_commands
 
 
+def test_image_cmd_malformed(builder_version, servicer, client, tmp_path_with_content):
```

Comment:
Don't think these fixtures are needed FyI since the error should get raised immediately.

Reply:
Thanks. Too fast on the copy pasta there.